protocol/src/main/java/org/fidoalliance/fdo/protocol/HttpClient.java

```diff
@@ -94,15 +94,32 @@ protected void sendMessage() throws IOException {
 
       try (CloseableHttpClient httpClient = Config.getWorker(HttpClientSupplier.class).get()) {
 
+        MsgType msgId = getRequest().getMsgType();
+        HttpInstruction httpInstruction = getInstructions().get(index++);
         URIBuilder uriBuilder = new URIBuilder(
-            getInstructions().get(index++).getAddress());
+            httpInstruction.getAddress());
+
+        if (msgId == MsgType.TO1_HELLO_RV) {
+          if (!httpInstruction.isRendezvousBypass()) {
+            logger.info("RVBypass flag not set, Starting TO1.");
+          }
+          logger.info("TO1 URL is " + uriBuilder.toString());
```

Comment:
No logs for TO1 failure. 
Expected log message: "TO1 failed. Unable to onboard device. Exiting application."